redbot/cogs/audio/core/tasks/player.py

```diff
@@ -28,7 +28,7 @@ async def player_automated_timer(self) -> None:
                 if await self.bot.cog_disabled_in_guild(self, server):
                     continue
 
-                if [self.bot.user] == p.channel.members:
+                if not any(m for m in p.channel.members if not m.bot):
```

Comment:
I think this is easier to comprehend when reading the code:
```suggestion
                if p.channel.members and all(m.bot for m in p.channel.members):
```
It also fixes an issue with an empty sequence triggering this `if` (though it's probably a case that wouldn't be hit anyway and even if so, it would just be ignored later on).

Reply:
Fair point, I've committed the change